testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -729,6 +729,27 @@ public void testSelectInformationSchemaColumns()
                         "('views')");
     }
 
+    @Test
+    public void testShowInformationSchema()
```

Comment:
nit: may be `testShowCreateInformationSchema` and similarly below?

Reply:
dobe